specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2021-07-01-preview/virtualNetworks.json

```diff
@@ -0,0 +1,575 @@
+                            },
+                            "description": "Routes - Collection of routes contained within a route table.",
+                            "x-ms-identifiers": []
+                          }
+                        },
+                        "description": "RouteTablePropertiesFormat route Table resource."
+                      },
+                      "type": {
+                        "type": "string",
+                        "description": "Type - READ-ONLY; Resource type."
+                      }
+                    },
+                    "description": "RouteTable for the subnet"
+                  },
+                  "ipPools": {
+                    "type": "array",
+                    "description": "network associated pool of IP Addresses",
+                    "items": {
+                      "$ref": "#/definitions/IPPool"
+                    },
+                    "x-ms-identifiers": []
+                  },
+                  "vlan": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Vlan to use for the subnet"
+                  }
+                },
+                "description": "SubnetProperties - Properties of the subnet."
+              }
+            },
+            "description": "Subnet subnet in a virtual network resource."
+          },
+          "description": "Subnet - list of subnets under the virtual network",
+          "x-ms-identifiers": []
+        },
+        "provisioningState": {
+          "type": "string"
+        },
+        "resourceName": {
+          "type": "string",
+          "description": "name of the object to be used in moc"
+        },
+        "status": {
+          "type": "object",
+          "description": "MOCVirtualNetworkStatus defines the observed state of MOCVirtualNetwork"
+        }
+      },
+      "description": "MOCVirtualNetworkSpec defines the desired state of MOCVirtualNetwork"
+    },
+    "IPPoolInfo": {
+      "type": "object",
+      "properties": {
+        "used": {
+          "type": "string",
+          "readOnly": true,
+          "description": "no. of ip addresses allocated from the ip pool"
+        },
+        "available": {
+          "type": "string",
+          "readOnly": true,
+          "description": "no. of ip addresses available in the ip pool"
+        }
+      }
+    },
+    "IPPool": {
+      "type": "object",
+      "properties": {
+        "ipPoolType": {
+          "type": "string",
+          "enum": [
+            "vm",
+            "vippool"
+          ],
+          "x-ms-enum": {
+            "name": "IPPoolTypeEnum",
+            "modelAsString": false
+          },
+          "description": "ip pool type"
+        },
+        "start": {
+          "type": "string",
+          "description": "start of the ip address pool"
+        },
+        "end": {
+          "type": "string",
+          "description": "end of the ip address pool"
```

Comment:
>end of the ip address pool [](http://example.com/codeflow?start=26&length=26)

First IP/last IP of the pool is slightly more specific language than Start/End of pool.